logistics_project/apps/malawi/warehouse/runner.py

```diff
@@ -444,7 +464,7 @@ def update_consumption(report_period):
 
         start_time = max(report_period.supply_point.created_at,
                          report_period.window_date)
-        if start_time < report_period.period_end and report_period.supply_point.supplies(p):
+        if start_time < report_period.period_end and p.pk in products_managed:
```

Comment:
i confirmed this is an equivalent check, and it's faster since we just get the managed product set once instead of looking up to the many-to-many table each time